docs/en/stack/security/securing-communications/tutorial-tls-internode.asciidoc

```diff
@@ -50,61 +127,51 @@ For example, run the following commands on each node:
 <1> If the {es} keystore already exists, this command asks whether you want to
 overwrite it. You do not need to overwrite it; you can simply add settings to
 your existing {es} keystore.
+// end::secure-passwords[]
 
-You are prompted to supply the password value. As you saw in the previous step,
-we are using the same file for both the transport TLS keystore and truststore,
-therefore you supply the same password for both of these settings.
+You are prompted to supply the password that you created for the `node-1.p12`
+file. We are using this file for both the transport TLS keystore and truststore,
+therefore supply the same password for both of these settings.
 --
 
-. Start each {es} node. For example, if you installed {es} with a `.tar.gz`
-package, run the following command from each {es} directory:
+. {ref}/starting-elasticsearch.html[Start {es}].
 +
 --
+For example, if you installed {es} with a `.tar.gz` package, run the following
+command from the {es} directory:
+
 ["source","sh",subs="attributes,callouts"]
 ----------------------------------------------------------------------
 ./bin/elasticsearch
 ----------------------------------------------------------------------
-
-See {ref}/starting-elasticsearch.html[Starting {es}].
 --
 
-. (Optional) Restart {kib}. For example, if you installed 
-{kib} with a `.tar.gz` package, run the following command from the {kib} 
-directory:
+. Create passwords for the built-in users and . 
 +
 --
-["source","sh",subs="attributes,callouts"]
-----------------------------------------------------------------------
-./bin/kibana
-----------------------------------------------------------------------
+NOTE: If you already created these users for other tutorials, you can skip this
```

Comment:
This terminology is misleading. These users always exist, the command simply sets their passwords.
```suggestion
NOTE: If you already configured passwords for these users for other tutorials, you can skip this
```